lib/get-page-data.js

```diff
@@ -6,16 +6,21 @@ function getPageData(html) {
     const $ = cheerio.load(html);
     const $title = $('h1').first();
     const $subtitle = $('p').first();
+    const $hr = $('hr').last();
+    const footerContent = $hr.nextAll();
+    const footer = $(footerContent).map((i, elem) => $(elem)).get().join('');
```

Comment:
[.get()](https://github.com/cheeriojs/cheerio#get-i-) retrieves the DOM elements matched by the Cheerio object. That's what we want, right?